components/crossplane-control-plane/base/cronjob.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: namespace-claim-cleaner
+  namespace: crossplane-control-plane
+spec:
+  schedule: "0 4 * * *" # every day at 4AM UTC
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: namespace-claim-cleaner
+              image: bitnami/kubectl:latest
```

Comment:
We shouldn't use `docker.io` due to security risks and rate limiting issues. Pulling the `latest` tag is also generally ill advised for production code - we should pull by a more specific tag and digest whenever possible. Please use the [appstudio-utils](https://quay.io/repository/konflux-ci/appstudio-utils?tab=tags) image instead which has `kubectl` and `jq`/`yq` baked in.